PrmPkg/PrmConfigDxe/PrmConfigDxe.c

```diff
@@ -100,12 +100,15 @@ SetRuntimeMemoryRangeAttributes (
 
     Status2 = EFI_NOT_FOUND;
     Status  = gDS->GetMemorySpaceDescriptor (RuntimeMmioRanges->Range[Index].PhysicalBaseAddress, &Descriptor);
-    if (!EFI_ERROR (Status) &&
-        (
-         ((Descriptor.GcdMemoryType != EfiGcdMemoryTypeMemoryMappedIo) && (Descriptor.GcdMemoryType != EfiGcdMemoryTypeReserved)) ||
-         ((Descriptor.Length & EFI_PAGE_MASK) != 0)
-        )
-        )
+    if (!EFI_ERROR (Status) && (Descriptor.GcdMemoryType == EfiGcdMemoryTypeNonExistent)) {
+      // The MMIO range is visible but has not been added to the memory space treat the region as not found.

```

Comment:
Can we improve this comment please? What does 'visible' mean in this context?

And punctuation marks are free ...

Reply:
I cherry-picked this from a downstream commit. I will update the comments

Reply:
Updated.